modules/imgproc/src/colormap.cpp

```diff
@@ -499,11 +498,8 @@ namespace colormap
             CV_Error(Error::StsAssert, "cv::LUT only supports tables of size 256.");
         Mat src = _src.getMat();
         // Return original matrix if wrong type is given (is fail loud better here?)
```

Comment:
@lvv , yes, it can be closed. Try to merge __your__ _master_ branch into this one and then squash all commits into one.